src/test/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicatorTest.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (c) 2021 AVI-SPL Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.aver.ptz;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.ReplyPacket;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.Command;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.CommandType;
+
+/**
+ * Unit test for AverPTZ Communicator
+ * Send success packet by override method read and doneReading
+ *
+ * @author Harry
+ * @version 1.0
+ * @since 1.0
+ */
+public class AverPTZCommunicatorTest {
+	AverPTZCommunicator averPTZCommunicator;
+
+	@Before
+	public void setUp() throws Exception {
+		averPTZCommunicator = new AverPTZCommunicator();
+		averPTZCommunicator.setHost("172.31.254.204");
+		averPTZCommunicator.setPort(52381);
+		averPTZCommunicator.setLogin("root");
+		averPTZCommunicator.setPassword("1234");
+		averPTZCommunicator.init();
+		averPTZCommunicator.connect();
+	}
+
+	@After
+	public void destroy() {
+		averPTZCommunicator.disconnect();
+	}
+
```

Comment:
It would be better if you can add the test case for Backlight, WBMode and PanTiltSlowMode

Reply:
Oh I see, thank you, I forgot  that